test/fixtures/sub-process.js

```diff
@@ -4,7 +4,15 @@ var Insight = require('../../lib');
 var insight = new Insight({
 	packageName: 'yeoman',
 	packageVersion: '0.0.0',
-	trackingCode: 'GA-1234567-1'
+	trackingCode: 'GA-1234567-1',
+	config: {
```

Comment:
I've updated this to fix merge conflicts (thanks, @sindresorhus), but don't see how to fulfill this request, given that we're dealing with child processes. If you're going to block on it, then I suppose the PR should be closed (unless someone else has an idea).